poker-pals/src/components/Reports.vue

```diff
@@ -131,24 +133,26 @@
       }
     }
   },
-  beforeCreate(){
-    this.socket = io("http://localhost:3000");
-
-    // Request reports from the server
-    this.socket.emit('request reports');
-
-    // Receive the reports and update the data and column arrays
-    this.socket.on('receive reports', data => {
-      this.data = data.reports;
-      this.columns = data.gridColumns;
-    });
-
-    if (this.columns) {
-      let sortOrders = {};
-      this.columns.forEach(function (key) {
-        sortOrders[key] = 1;
-      })
-      this.sortOrders = sortOrders;
+  mounted(){ // switched to mounted, props are not yet set in beforeCreate(), 
+    if(!this.authenticated){
+        this.$router.replace({ name: "Login" });
```

Comment:
Once again, I thought I was going to add this to the reports page...

Reply:
looking back in the trello i guess it should of been made more clear on the scope of each task. for my card 'implement single page navigation', i assumed it was for all components

Reply:
That's okay, but please let me know next time so we don't end up doing the same thing twice